MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -83,34 +83,6 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
         return nil
```

Comment:
Given that multiple access tokens can now exist for a single account, we should:
- get rid of the comment at line 72 because it states something that it isn't true anymore
- retrieve only IdToken  in the`retrieveUserAccountResult` function. We will retrieve Access and Refresh token only when the user requests for it.
- Now we can get rid of the `MSALNativeAuthTokens` class

Reply:
@nilo-ms , I already removed handleMSIDTokenResponse, handleTokenResponse, the comment on line 72 and retrieved only IdToken in `theretrieveUserAccountResult` method. I do agree that we could get rid of `MSALNativeAuthTokens` class and refactor our cache accessor, but this will be a big change, it affects dozens of existing unit test, and it seems beyond the scope of this feature. I suggest creating a PBI in our backlog for this refactor.

Reply:
I believe that retrieving only IdToken in the `retrieveUserAccountResult` and as consequence removing the `MSALNativeAuthTokens` is part of this ticket. This feature is to allow external developer retrieving and storing multiple access tokens in the SDK. So, this means that the user needs to specify scopes to retrieve access token. This PR makes the retrieval of the access token and refresh token as part of the account retrieval not needed anymore. So, in my opinion, this code needs to be removed in this PR.

Reply:
Done.